src/components/LineInput/index.tsx

```diff
@@ -13,9 +13,9 @@ import "./LineInput.css";
 import {
     activeInput,
     caret,
```

Comment:
Let's sort these imports.

```suggestion
    accordionButton,
    accordionText,
    accordionTitle,
    activeInput,
    caret,
```